File: C Programming/Assignments/Assignment 6/function with return type and with para/Assignment 1/Q6/Q6.c
//Write a program to check whether a given character is uppercase or lowercase.

#include<stdio.h>

char checkUpperLower(char);
void main(){
	
	//variable initialization
	char ch = 'a';

	char res = checkUpperLower(ch);
	
	if(res=='u')
	   printf("Uppercase");
	else if(res=='l')
	   printf("Lowercase");
	   
}

char checkUpperLower(char ch){

	//applying condition
	if(ch>='A' && ch<='Z')
		return 'u';
	else if(ch>='a' && ch<='z')
		return 'l';
	else
		printf("Not a valid character Input please try again.");
		
}

File: FirstBit Workspace/C Programming/Assignments/java_to_c_connectivity/Assignment 1/Q6/Q6.java
//Write a program to check whether a given character is uppercase or lowercase.

public class Q6 {
    public static void main(String[] args) {
        // variable initialization
        char ch = 'a';

        // applying condition
        if (ch >= 'A' && ch <= 'Z')
            System.out.println("Given character -> (" + ch + ") is an Uppercase character");
        else if (ch >= 'a' && ch <= 'z')
            System.out.println("Given character -> (" + ch + ") is a lowercase character");
        else
            System.out.println("Not a valid character Input please try again.");
    }
}
